NodeMCU-Home-Automation-Sensor.ino

```diff
@@ -78,14 +77,13 @@ PubSubClient client(espClient);
 #ifdef OLED_SPI
   SSD1306Spi  display(D10, D9, D8); //RES, DC, CS (SPI Initialization)
 #else
-  SSD1306     display(0x3c, 3 /*D9*/, 1 /*D10*/); //0 /*D3*/, 2 /*D4*/); //SDA, SCL (I2C Initialization)
+  //SSD1306     display(0x3c, 3 /*D9*/, 1 /*D10*/); //
+  SSD1306     display(0x3c, 0 /*D3*/, 2 /*D4*/); //Used D3-D4 so USB serial monitor works, since it uses D9-D10
```

Comment:
Good catch, but it brings with it more changes (namely the readme table/diagram, and SPI line above should also be changed). Since all of my sensors are already wired up to use the D9/D10, the Youtube videos reference those pins as well, and I don't use the serial monitor once they're deployed, I'm going to keep the existing values and provide a comment/warning to the end user to change the pins if they need the USB Serial monitor to work.